test.js

```diff
@@ -44,11 +44,6 @@ test('main', t => {
 	t.is(pupa('{{0}}{{1}}', ['<br>yo</br>', '<i>lol</i>']), '&lt;br&gt;yo&lt;/br&gt;&lt;i&gt;lol&lt;/i&gt;');
 });
 
-test('do not match non-identifiers', t => {
-	const fixture = '"*.{json,md,css,graphql,html}"';
-	t.is(pupa(fixture, []), fixture);
-});
```

Comment:
I've gotten kind of burned out on my other projects. I foolishly tried to parse `error.stack`, but the more I learn about what it can contain, the more I think it's likely a total loss in the general case.

Anyway, that is to say that I need to find a way to feel like I'm making useful contributions again, and I'd be willing to help you get this merged. But I'd need to resolve this discussion first. How do you feel about the trade off I made?